### DISCUSSION TITLE
warning "insufficient" without specific details

### DISCUSSION THREAD

**Discussion by xerc** [okd-project/okd#877] [Q&A]
![Bildschirmfoto 2021-09-17 um 15 46 20](https://user-images.githubusercontent.com/1372590/133807323-cab27dc8-5b09-4428-b5d4-2992356e4159.png)
--
does anyone know (intern) details whats "missing" or is there a bash-script for checking servers before booting from USB:ISO?

**Comment by bdlink** [comment]
Since I cannot find the minimum node requirements, there is perhaps a documentation issue here, and I agree the message is not helpful.

My understanding is that 16 GB of memory is needed for each node (certainly the control nodes, but possibly worker nodes as well).  As the versions move forward, I could see this getting tight.

Hope this helps until an authoritative answer comes forth.



On Sep 17, 2021, at 8:21 AM, Marcus Förster ***@***.***> wrote:

﻿
CAUTION: This email originated from outside of BCIT. Do not click links or open attachments unless you recognize the sender and know the content is safe.


[Bildschirmfoto 2021-09-17 um 15 46 20]<https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fuser-images.githubusercontent.com%2F1372590%2F133807323-cab27dc8-5b09-4428-b5d4-2992356e4159.png&data=04%7C01%7Cbruce_link%40bcit.ca%7Cc084e6e68597417dbb9908d979eedeed%7C8322cefd0a4c4e2cbde5b17933e7b00f%7C0%7C0%7C637674889106992740%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=8ax9vgQEzvWSRSoQfhUywPwKvbC1gIZ%2BnB%2FY6rGnAD4%3D&reserved=0>

does anyone know (intern) details whats "missing" or is there a bash-script for checking servers before booting from USB:ISO?

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub<https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fopenshift%2Fokd%2Fdiscussions%2F877&data=04%7C01%7Cbruce_link%40bcit.ca%7Cc084e6e68597417dbb9908d979eedeed%7C8322cefd0a4c4e2cbde5b17933e7b00f%7C0%7C0%7C637674889107002701%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=aFQBAitSCrzSnZrNp3gNr0QKOeJSVijnZnlK1PSOP30%3D&reserved=0>, or unsubscribe<https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAAEBGE36UZSAWU2F46MGEFDUCNMI3ANCNFSM5EHT7WGQ&data=04%7C01%7Cbruce_link%40bcit.ca%7Cc084e6e68597417dbb9908d979eedeed%7C8322cefd0a4c4e2cbde5b17933e7b00f%7C0%7C0%7C637674889107002701%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=tGlafj3EZ2p1u8n8Ii2nLYw60RUf%2BbSfp6QDdvrF0Fg%3D&reserved=0>.
Triage notifications on the go with GitHub Mobile for iOS<https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fapps.apple.com%2Fapp%2Fapple-store%2Fid1477376905%3Fct%3Dnotification-email%26mt%3D8%26pt%3D524675&data=04%7C01%7Cbruce_link%40bcit.ca%7Cc084e6e68597417dbb9908d979eedeed%7C8322cefd0a4c4e2cbde5b17933e7b00f%7C0%7C0%7C637674889107002701%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=EdJwT4xFhnR7RfP8jaOt6mq0ogSF7BdFoMIvvXYXdik%3D&reserved=0> or Android<https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fplay.google.com%2Fstore%2Fapps%2Fdetails%3Fid%3Dcom.github.android%26referrer%3Dutm_campaign%253Dnotification-email%2526utm_medium%253Demail%2526utm_source%253Dgithub&data=04%7C01%7Cbruce_link%40bcit.ca%7Cc084e6e68597417dbb9908d979eedeed%7C8322cefd0a4c4e2cbde5b17933e7b00f%7C0%7C0%7C637674889107012655%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=UqGtlHs7cSs2F%2FphaxIZmr70%2BwW0xU%2BeFDadOhFKIds%3D&reserved=0>.

**Comment by xerc** [comment]
_as declared within the WebInterface_
**Minimum hardware requirements**
Worker hosts must have at least 2 CPU cores, 8.00 GiB of RAM, and 120 GB of filesystem storage.

**Reply by xerc** [reply]
fyi/ just started the machine with 64 GB ECC-RAM with the same "result"

**Comment by xerc** [comment]
btw/ this board has (only) iLO3
